hawk/app/assets/javascripts/module/resources.js

```diff
@@ -114,7 +114,7 @@ $(function() {
             } else if (row.type == "clone") {
               return __("Clone");
             } else if (row.template != null) {
-              return '<a href="' + Routes.agent_path("@" + row.template) + '" data-toggle="modal" data-target="#modal-lg">' + "@" + row.template + '</a>';
+              return '<a href="' + Routes.agent_path({id: "@" + row.template}) + '" data-toggle="modal" data-target="#modal-lg">' + "@" + row.template + '</a>';
```

Comment:
Line is too long.<br>Mixed double and single quotes.<br>Identifier 'agent_path' is not in camel case.